opl-secret-ballot/.gitignore

```diff
@@ -1,2 +1,7 @@
 node_modules/
 *.log
+.cache
+.bashrc
```

Comment:
Does `.bashrc` appear because the folder behaves as a home dir inside the docker?

Reply:
Yes that's correct.